students/randi_peterson/session06/mailroom.py

```diff
@@ -0,0 +1,109 @@
+import os.path as path
+
+def thank_you(donor_dict):
+    #Record single donation and thank donor
+    fullname = input('What is the full name of the donor? Type list to see donors. Type exit to quit task \n')
+    give_thanks(donor_dict,fullname)
+
+def give_thanks(donor_dict, fullname):
+    #Executes the thank you note, isolating data from input
+    if fullname.lower() == 'list':
+        print(list(donor_dict.keys()))
+    elif fullname.lower() == "exit":
+        return None
+    else:
+        new_donation = float(input('What is the value of the donation (numbers only)?\n'))
+        add_to_list(donor_dict,fullname,new_donation)
+        #Print the thank you email
+        print(thank_you_text(fullname))
+
+def add_to_list(donor_dict,fullname,new_donation):
+    # Find donor in list or create a new one
+    if fullname not in donor_dict:
+        donor_dict[fullname] = []
+    donor_dict[fullname].append(new_donation)
+
+def calculate_report(name,donor_dict):
```

Comment:
why pass these two params when you can pass in the value in a dictionary - you don't actually use the name